src/test/java/com/speaker/service/AccountServiceImplTest.java

```diff
@@ -14,16 +17,23 @@
 import org.mockito.junit.jupiter.MockitoExtension;
 
 import java.util.List;
+import java.util.Optional;
 
 import static com.speaker.utils.AccountDTOGenerator.*;
 import static com.speaker.utils.AccountGenerator.*;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.core.Is.is;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
 
 @ExtendWith(MockitoExtension.class)
 class AccountServiceImplTest {
-
+    final String IVAN_MAZUR = "Ivan Mazur";
+    final String VICTOR_MAZUR = "Victor Mazur";
+    final String IVAN = "Ivan";
+    final String MAZUR = "Mazur";
+    final String VICTOR = "Victor";
```

Comment:
missed: access modification and `static`

Reply:
modification and static added. Changes made